### DISCUSSION TITLE
PCBWay tutorial

### DISCUSSION THREAD

**Discussion by vicsidious** [pmgducati/Blue-Retro-AIO-Units#63] [Q&A] [answered]
Hi! What you did is an AMAZING job! All those pcbs are designed so beautifully, it looks very easy to create new cables for blueretro now! 

But here's the deal: I know how to properly solder and can do intermediate to advanced mods on about anything, but it's my first time building PCBs and I found out that the easiest way would be to use a service like PCBWay to build it for me at low costs (thank you Linus sponsors!). Unfortunately, there seems to be a handful of options that I cannot find the correct value for! Can I leave all option, like FR4-TG, Track Spacing and Hole Size at a default value, upload the Gerber files and hope for it to work, or is there something particular I should setup before ordering? I'll attach an image of these options and their default values. 

I understand this might seems like a pretty stupid question, but hey, you gotta start somewhere, right? 
![pcb](https://user-images.githubusercontent.com/115555660/219076794-c23bbf68-bfaf-4891-aa32-251e4a47e157.JPG)

**Comment by pmgducati** [comment, marked as answer]
Hi @vicsidious ,
For the most part the default selections work for this design, almost any change that you make (with the exception of color generally) you are going to incur cost increases.  Most sections are determined by the design, the exceptions are the color of the PCB/Silkscreen and the surface finish. (although the surface finish can be defined in the design also.)  Generally when creating the package for the PCB, one would either add any deviations from the default to the BOM or via a packet that would reside with the gerber file.

One thing that I would recommend is checking out JCLPCB.  Recently they have been cheaper than PCB way and I have never received a faulty PCB from them where I have had occasional issues with PCBway (to be fair though it has been about 2 years since I have ordered from PCBWay as they cant beat JLCPCB prices)

Let me know if you have any other questions.

**Reply by vicsidious** [reply]
Hey! Thanks a lot for the quick reply! I can't wait to get this project started! I'll take a look at JLCPCB right away! 
I'll poste pictures when the boards are created and populated :) I think I'll go with a blue board, because, you know... it's not called purpleretro! ;)

**Reply by pmgducati** [reply]
Glad I could help, when I am working on designing the PCB, I will use the colors as a visual way for me to recognize the rev.  

I'm excited to see your PCBs! Post pictures when you get them.  

Quick question, what systems are you planning on getting?  There are a few consoles I don't have and have not been able to test (Check out the issues).  If you get any of them any feedback would be greatly appreciated!

**Reply by vicsidious** [reply]
I plan to use it on Neo Geo and Turbografx 16, as I got all the other cables from other sources! I might make one for pcengine, since a friend of mine has one!
Will post everything as soon as it is done! Also, I might only use acrylic to encase the pcb, not actual printed case. Do you think I might get some issues? 🤔
